jobs/build.Jenkinsfile

```diff
@@ -230,9 +230,38 @@ lock(resource: "build-${params.STREAM}") {
 
         // fetch from repos for the current build
         stage('Fetch') {
+            def overrides_fetch_param = ""
+            # Dont run this for production builds
+            recent_commits_to_lockfiles() {
```

Comment:
let's move this function out of the code path here (so it doesn't cloud reading the code). For example [look](https://github.com/coreos/fedora-coreos-pipeline/blob/a31b30aaaeac99a562c6971a4a29c598b61f2231/jobs/bump-lockfile.Jenkinsfile#L48) at how `getLockfileInfo()` is above the main set of code in bump-lockfile.Jenkinsfile.

Reply:
Thanks for all above @dustymabe!